openfisca_core/entities.py

```diff
@@ -53,6 +53,23 @@ def __call__(self, variable_name, period = None, options = [], **parameters):
         else:
             return self.simulation.calculate(variable_name, period, **parameters)
 
+    # JSON
+
+    @classmethod
+    def to_json(cls):
+        entity_dict = {
+            'key': cls.key,
+            'label': cls.label,
+            'plural': cls.plural,
+            }
+        if cls.is_person:
+            entity_dict['isPersonsEntity'] = cls.is_person
```

Comment:
Unless needed for backward compat, I'm in favour of enhancing the representation  😃